installer/windows/agent-installer.ps1

```diff
@@ -1,4 +1,8 @@
 if ($env:PROCESSOR_ARCHITECTURE -eq "AMD64") { $architecture = "x86_64" }
 if ($env:PROCESSOR_ARCHITECTURE -eq "ARM64") { $architecture = "arm64" }
 if ([string]::IsNullOrEmpty($architecture)) { throw "Architecture $env:PROCESSOR_ARCHITECTURE is not supported yet, please create a ticket in openbas github project" }
-Stop-Service -Force -Name "OBAS Agent Service"; Invoke-WebRequest -Uri "${OPENBAS_URL}/api/agent/package/openbas/windows/${architecture}" -OutFile "openbas-installer.exe"; ./openbas-installer.exe /S ~OPENBAS_URL="${OPENBAS_URL}" ~ACCESS_TOKEN="${OPENBAS_TOKEN}"; Start-Sleep -Seconds 1.5; rm -force ./openbas-installer.exe;
\ No newline at end of file
+
+$directory = (Get-Location).Path
```

Comment:
I think its more about the agent to adapt the rights of the implant directory.

Reply:
Sure, It was my first try (agent modifies rights on directory) but I had some difficults writing the code so for that I include it here (I'm going to ask to modify it), ty.